app/api/data.py

```
@@ -0,0 +1,1157 @@
+router = APIRouter(prefix="/data", tags=["data"])
+
+
+@router.get("/teams", response_model=List[Team])
+async def get_teams(
+    conference: Optional[str] = None,
+    division: Optional[str] = None,
+    include_stats: bool = Query(False, description="Include current season stats"),
+    include_sentiment: bool = Query(
+        False, description="Include current sentiment data"
+    ),
+    db=Depends(get_database),
```

Ruff (0.13.3)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)